README.md

```diff
@@ -198,18 +198,18 @@ For \[on-premises\] Team Foundation Server, the _User name_ can be specified in
 1. `EXAMPLE-DOMAIN\user`
 2. `user@domain.example.com`
 
-#### Visual Studio Team Services (Team Services, previously known as Visual Studio Online)
+#### Azure DevOps (Azure DevOps, previously known as Visual Studio Online)
```

Comment:
Change to
`Azure DevOps (previously known as Visual Studio Team Services or VSTS or Visual Studio Online or VSO)`

;)